content/docs/welcome-guide/requirements.md

```diff
@@ -44,6 +44,7 @@ or later is required.
 ### Microsoft Visual Studio
 
 + [Microsoft Visual Studio 2019](https://docs.microsoft.com/en-us/visualstudio/releases/2019/release-notes) version **16.9.2** through version **16.11.x** are supported with O3DE. You can use any Microsoft Visual Studio license, including the Community edition.
++ [Microsoft Visual Studio 2022](https://docs.microsoft.com/en-us/visualstudio/releases/2022/release-notes) version **17.3** through version **17.3.x** are supported with O3DE. You can use any Microsoft Visual Studio license, including the Community edition. **A minimum of CMake 3.21.7 or above is required to build with Visual Studio 2022**.
```

Comment:
Please also update the text about the redistributable version. It currently reads:

"You will also need the Visual C++ Redistributable for Visual Studio 2019."

It should read "[...] for Visual Studio 2022." to match the existing link.

Although is this section regarding the redistributable really needed at all? I thought the redistributable is included with Visual Studio installs. Or are there scenarios where this wouldn't get installed?

Reply:
Well when the user installs the Desktop development with C++ workload for Visual Studio, the redistributable gets installed as a required component. 
![image](https://user-images.githubusercontent.com/56135373/187558147-173b1acd-f26e-4366-8c42-51920151cba1.png)

Reply:
Exactly, so if they're following our earlier instructions, which say to use this workload, it would already be installed.

Reply:
I'll see about rewording the redistributable instructions (or even remove it). It's not likely that there's a scenario where they don't have Visual Studio installed, as it's a requirement to build the project.